metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -182,26 +179,22 @@ object ServerCommands {
        |QuickPick will show up only if more than one result is found.
        |
        |Note: document in json argument must be absolute path.
-       |
-       |Arguments:
-       |```json
-       |{
+       |""".stripMargin,
+    """|{
```

Comment:
I still think we need to figure out a way for this to be a code block potentially. I find the arguments here pretty hard to read.

<img width="860" alt="Screenshot 2020-04-16 at 14 56 00" src="https://user-images.githubusercontent.com/13974112/79458973-e0e2e200-7ff2-11ea-99dd-3427037823c1.png">